state-synchronizer/src/tests/integration_tests.rs

```diff
@@ -224,18 +303,19 @@ impl SynchronizerEnv {
         let mut network_handles = vec![];
         if mock_network {
             let networks = if role.is_validator() {
-                vec![NetworkId::Validator, NetworkId::vfn_network()]
+                vec![VALIDATOR_NETWORK.clone(), VFN_NETWORK.clone()]
             } else {
                 vec![
-                    NetworkId::vfn_network(),
-                    NetworkId::Private("second".to_string()),
-                    NetworkId::Public,
+                    VFN_NETWORK.clone(),
```

Comment:
this feels kinda bad, but I guess symmetry?

Reply:
Yeah, I don't think it's nice either to be honest.. I'll try and revisit it once I've fried the bigger fish 🐟